yojenkins/cli/cli_build.py

```diff
@@ -350,4 +352,24 @@ def rebuild(profile: str, token: str, job: str, number: int, url: str, latest: b
         data = yj_obj.build.rebuild(build_url=url, job_url=job, build_number=number, latest=latest)
     else:
         data = yj_obj.build.rebuild(build_url=url, job_name=job, build_number=number, latest=latest)
-    click.secho(f'success. queue number: {data}', fg='bright_green', bold=True)
+
+    if not show_logs:
+        click.secho(f'success. queue number: {data}', fg='bright_green', bold=True)
+        return
+
+
+    while True:
+        click.echo(f"Build is queued with ID {data}. Waiting ...")
+        time.sleep(5)
+        queue_data = yj_obj.job.queue_info(build_queue_number=data)
+        if "executable" in queue_data:
+            break
+
+    job_number = queue_data["executable"]["number"]
+    click.echo(f"Job is running with ID {job_number}. Output below:")
+    yj_obj.build.logs(build_url=None,
+                job_url=queue_data["jobUrl"],
+                build_number=job_number,
+                follow=True)
+
+
```

Comment:
Good to run `pre-commit run` on all staged files. I'm not sure if you glanced at it, there is a little blurb here:
https://www.yojenkins.com/contribute/#making-changes

(By the way, none of these contribution steps are set in stone, just guidelines)